src/algebra/ordered_monoid.lean

```diff
@@ -75,6 +75,70 @@ ordered_comm_monoid.mul_le_mul_left a b h c
 lemma mul_le_mul_right' (h : a ≤ b) (c) : a * c ≤ b * c :=
 by { convert mul_le_mul_left' h c using 1; rw mul_comm }
 
+@[to_additive]
+lemma mul_lt_of_mul_lt_left (h : a * b < c) (hle : d ≤ b) : a * d < c :=
+(mul_le_mul_left' hle a).trans_lt h
+
+@[to_additive]
+lemma lt_of_mul_lt_left (h : a * b < c) (hle : 1 ≤ b) : a < c :=
+by simpa only [mul_one] using mul_lt_of_mul_lt_left h hle
```

Comment:
I'd recommend you add these two lemmas:
```lean
@[to_additive]
lemma le_mul_right_of_one_le (hle : 1 ≤ b) : a ≤ a * b :=
by simpa only [mul_one] using mul_le_mul_left' hle a

@[to_additive]
lemma le_mul_left_of_one_le (hle : 1 ≤ b) : a ≤ b * a :=
by simpa only [one_mul] using mul_le_mul_right' hle a
```
and then this proof (and the many like it) becomes
```suggestion
(le_mul_right_of_le hle).trans_lt h
```

Reply:
These lemmas already exist as `le_mul_of_one_le_right'` and `le_mul_of_one_le_left'` below (old line 90, new line 154). But I can rework my proofs to be derived from those if you'd like.

Reply:
I think it makes sense to use those in your proofs, and hasn't spotted them below - thanks